site/app/views/submission/HomeworkView.php

```diff
@@ -47,15 +47,24 @@ public function showGradeableError($gradeable) {
      * TODO: BREAK UP THIS FUNCTION INTO EASIER TO MANAGE CHUNKS
      *
      * @param Gradeable $gradeable
-     * @param int       $days_late
+     * @param int       $late_days_use
+     * @param int       $extensions
      *
      * @return string
      */
-    public function showGradeable($gradeable, $days_late) {
+    public function showGradeable($gradeable, $late_days_use, $extensions) {
+        $return = "";
+        if($extensions != 0){
```

Comment:
We probably want to have it be `if ($extensions > 0)` (as it should never be the case this is negative).